src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1021,9 +1021,8 @@ def load_arguments(self, _):
     for scope in ['vm create', 'vmss create']:
         with self.argument_context(scope) as c:
             c.argument('identity_role', options_list=['--role'], arg_group='Managed Service Identity',
-                       help='Role name or id the system assigned identity will have. '
-                            'Please note that the default value "Contributor" will be removed in the future version 2.35.0, '
-                            "so please specify '--role' and '--scope' at the same time when assigning a role to the managed identity")
+                       help="Role name or id the system assigned identity will have. "
+                            "Please specify '--role' and '--scope' at the same time when assigning a role to the managed identity")
```

Comment:
How about we put this in the command help? Otherwise, we have to have this text for both `--scope` and `--role`.

Reply:
@jiasli Since `az vm create` and `az vmss create` commands have too many other parameters and examples, I think it might be a good idea to add the description to both `--scope` and `--role` parameters

Reply:
Or maybe we can not include this information in the help description, because we already have verification logic that will tell users that those two parameters need to be passed in at the same time

Reply:
Duplication is bad. `ad sp create-for-rbac` puts this information in command help:

https://github.com/Azure/azure-cli/blob/b86a4ed33317e9b218c84d4c96498f75235df996/src/azure-cli/azure/cli/command_modules/role/_help.py#L392-L393

But I am open to this topic. 😉

Reply:
The situation of these two commands may be different, `az vm create` command has nearly 100 parameters and too much help information  😂.
Since the `--scope` and `--role` are only one of many usage scenarios, I'm not sure whether it's appropriate to put them in a global position in this case. And the help information printed by this command is too long, so users may not notice the description from the command level every time

In order to avoid duplicated prompts, I personally think maybe I can remove this information in the help description, because we already have the verification to tell users that `--scope` and `--role` parameters need to be passed in at the same time. What do you think of this idea?